components/infobox/wikis/trackmania/infobox_league_custom.lua

```diff
@@ -116,6 +122,9 @@ function CustomLeague:defineCustomPageVariables(args)
 	Variables.varDefine('date', self.data.endDate)
 	Variables.varDefine('sdate', self.data.startDate)
 	Variables.varDefine('edate', self.data.endDate)
+
+	Variables.varDefine('circuit', self.data.circuit)
+	Variables.varDefine('circuittier', self.data.circuittier)
```

Comment:
```suggestion
```
don't see any usage on the wiki for the wiki variables
if there is planned usage for them let us know that
mind this is only about the wiki variables not about storage in lpdb
wiki variables are used to pass down the information on the same page the infobox is set and nowhere else

Reply:
Don't think it will be used either. Removed